PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+/// The Bar class
+public class Bar {
+    var yValues: [Double]
+    var label: String
+    var color: [UIColor]?
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    /// - parameter color: provide a color base on the class UIColor
+    public init(_ yValues: [Double], label: String, color: [UIColor]) {
+        self.yValues = yValues
+        self.label = label
+        self.color = color
+    }
+
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    public init(_ yValues: [Double], label: String) {
+        self.yValues = yValues
+        self.label = label
+    }
+}
+
+/// PredixBarChartView -- Bar Chart
+open class PredixBarChartView: BarChartView {
+    /// :nodoc:
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialize()
+    }
+
+    /// :nodoc:
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initialize()
+    }
+
+    /// Predix Mobile Bar chart initial values
```

Comment:
For easier reading of the code, organize your properties and methods by type and access.
Putting // MARK: <description> comments at the heads of these groups helps organize the code too. See the other Views for examples.

Reply:
Added MARK to all of the PredixBarChart files